### PR TITLE
add a workflow to release wamr-lldb on Ubuntu

### PR DESCRIPTION
**Pull request by lum1n0us**
It takes 2h 10m to do a compilation on ubuntu-2004 runner after applying *Ninja* and *LLD*.

### CODE REVIEW

**Comment by lum1n0us** [comment]
Also fix a bug when compressing wamr-sdk

File: .github/workflows/build_wamr_lldb.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          cp build/bin/lldb* inst/bin
+          cp build/lib/liblldb*.so inst/lib
+          cp build/lib/liblldb*.so.* inst/lib
+          cp lldb/tools/lldb-vscode/package.json inst
+          cp -r lldb/tools/lldb-vscode/syntaxes/ inst
+        working-directory: core/deps/llvm-project
+
+      - name: compress the binary
+        run: |
+          tar czf warm-lldb-${{ inputs.ver_num }}-${{ inputs.runner }}.tar.gz inst
```
Comment (wenyongh) [inline .github/workflows/build_wamr_lldb.yml:102]: Should be wamr-lldb

File: .github/workflows/build_wamr_lldb.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          zip -r warm-lldb-${{ inputs.ver_num }}-${{ inputs.runner }}.zip inst
```
Comment (wenyongh) [inline .github/workflows/build_wamr_lldb.yml:103]: wamr-lldb

File: .github/workflows/build_wamr_lldb.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        working-directory: core/deps/llvm-project
+
+      - name: upload release tar.gz
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ inputs.upload_url }}
+          asset_path: core/deps/llvm-project/warm-lldb-${{ inputs.ver_num }}-${{ inputs.runner }}.tar.gz
```
Comment (wenyongh) [inline .github/workflows/build_wamr_lldb.yml:112]: wamr-lldb

File: .github/workflows/build_wamr_lldb.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+          asset_name: wamr-lldb-${{ inputs.ver_num }}-${{ inputs.arch }}-${{ inputs.runner }}.tar.gz
+          asset_content_type: application/x-gzip
+
+      - name: upload release zip
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ inputs.upload_url }}
+          asset_path: core/deps/llvm-project/warm-lldb-${{ inputs.ver_num }}-${{ inputs.runner }}.zip
```
Comment (wenyongh) [inline .github/workflows/build_wamr_lldb.yml:122]: wamr-lldb

File: .github/workflows/release_process.yml [hunk 5]
```diff
@@ -156,3 +156,13 @@ jobs:
     with:
       upload_url: ${{ needs.create_release.outputs.upload_url }}
       ver_num: ${{ needs.create_tag.outputs.new_ver }}
+
+  #
+  # WAMR_LLDB
+  release_wamr_lldb_on_ubuntu:
```
Comment (wenyongh) [inline .github/workflows/release_process.yml:162]: Shall we release wamr-lldb on macos?

Comment (lum1n0us) [reply .github/workflows/release_process.yml:162]: Amazon team is working on that

**Pull request merged**